quickjs.c

```diff
@@ -47230,6 +47230,19 @@ static JSValue JS_ThrowTypeErrorDetachedArrayBuffer(JSContext *ctx)
     return JS_ThrowTypeError(ctx, "ArrayBuffer is detached");
 }
 
+// #sec-get-arraybuffer.prototype.detached
+static JSValue js_array_buffer_get_detached(JSContext *ctx,
```

Comment:
Is this supported for SharedArrayBuffer? I suspect not, then we could omit the class_id parameter and check explicitly for ArrayBuffer, right?

Reply:
Yup, done